benchexec/tools/ultimateautomizer.py

```diff
@@ -25,18 +25,22 @@ class Tool(ultimate.UltimateTool):
     REQUIRED_PATHS = [
                   "artifacts.xml",
                   "AutomizerTermination.xml",
-                  "Automizer.xml",
                   "AutomizerWitnessValidation.xml",
+				  "Automizer.xml",
```

Comment:
Here and in many other lines of the `REQUIRED_PATHS` list in this file and the others there is inconsistent indentation. Python is quite sensitive about this, thus please make it consistent. Maybe you used tabs? (Spaces only, please)